src/converters.js

```diff
@@ -885,6 +885,8 @@ function injectViewList( modelItem, injectedItem, mapper, removePosition ) {
 		}
 	}
 
+	insertPosition = positionAfterUiElements( insertPosition );
```

Comment:
I just asked :P Out of curiosity, to gain knowledge and thus increase the bus factor ;)